### ISSUE TITLE
Not working web.conf samples

### ISSUE THREAD

**Issue by valib** [UniversalMediaServer/UniversalMediaServer#1448]
User send the problem http://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=12209 related to the subtitles but when checking his debug.log I found the problem with the "podcast" and "theonion". I can confirm that after testing this URLs don't work and should be either updated if possible or deleted from the web.conf

**Comment by Nadahar** [comment]
I've long thought that all those irrelevant links in ```web.conf``` should be removed/commented out. Most have no idea that they are there, and every time UMS is started on some computer these resources are loaded.

**Comment by valib** [comment]
@SubJunk do you have any idea how to solve it?

**Comment by SubJunk** [comment]
Yeah we should come up with some better samples, I'll try to find some

**Comment by SubJunk** [comment]
Also I'll look into adding automated tests around these feature to protect them from breaking again

**Comment by SubJunk** [comment]
There's an attempt at this in https://github.com/UniversalMediaServer/UniversalMediaServer/pull/1606
I couldn't find any working video streams so I commented them out, it is possible our mms support is broken
